components/compliance-service/api/tests/50_scheduler_spec.rb

```diff
@@ -156,8 +156,16 @@ def cleanup
   end
 
   it "creates new jobs as children of existing ones" do
-    sleep 70
-    all_jobs = GRPC jobs, :list, Jobs::Query.new()
+    counter = 0
+    while counter <= 10
```

Comment:
nitpick: you could avoid doing the manual counter iteration by using `Integer#times`: https://ruby-doc.org/core-2.5.0/Integer.html#method-i-times

Reply:
oh wow, i did not know that existed! im gonna give it a try, thanks!